api-dummy/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM php:7.1-fpm
+
+RUN mkdir -p /var/www && \
+    chown www-data:www-data /var/www
```

Comment:
a user to own the files making them not writable from `www-data` would be good, but can be easily postponed

Reply:
Sounds out of scope for this as a skeleton.